buoy/commands/exists.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"knative.dev/test-infra/pkg/gomod"
+)
+
+func addNextCmd(root *cobra.Command) {
+	var domain string
+	var release string
+	var verbose bool
+	var tag bool
+
+	var cmd = &cobra.Command{
+		Use:   "exists go.mod",
+		Short: "Determine if the release branch exists for a given module.",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			gomodFile := args[0]
+
+			var out io.Writer
+			if verbose {
+				out = os.Stderr
+			}
```

Comment:
I think cobra has decent support for managing output that you don't seem to be using.  See `cmd.SetOutput` and friends.  I wouldn't block on this, but might be worth follow-up

Reply:
then you learn!